kinder/pkg/kubeadm/config.go

```diff
@@ -74,10 +73,6 @@ func getKubeadmConfigVersion(kubeadmVersion *K8sVersion.Version) (string, error)
 	// v1alpha2 (that is Kuberntes v1.11.0) is out of support.
 	// v1alpha3 (that is Kuberntes v1.13.0) is out of support.
 
-	if kubeadmVersion.LessThan(constants.V1_15) {
-		return "v1beta1", nil
```

Comment:
@neolit123 the fact we are removing "v1beta1" from the kubeadm test grid is kind of worrying because it is still used in CAPI. 

We should consider adding a flag for forcing the config version to use and add dedicated tests for this (v1.16 with v1beta1, v1.16 with v1beta1 etc), but this is out of scope of this PR.

In the meantime, let's preserve int the code configTemplateBetaV1

Reply:
https://github.com/kubernetes/kubeadm/issues/2249

Reply:
i will commented on #2249